### PR TITLE
refactor(VStepper): convert to typescript

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
## Motivation and Context
It was my childhood dream

## How Has This Been Tested?
jest/lint

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VStepper/VStepper.ts [hunk 1]
```diff
@@ -30,16 +41,16 @@ export default {
...
-      content: [],
+      steps: [] as VStepperStepInstance[],
+      content: [] as VStepperContentInstance[],
       isReverse: false
     }
   },
 
   computed: {
-    classes () {
+    classes (): object {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VStepper/VStepper.ts]: Maybe use `{ [key:string]: boolean; }` type here?

File: packages/vuetify/src/components/VStepper/VStepper.ts [hunk 2]
```diff
@@ -30,16 +41,16 @@ export default {
 
   data () {
     return {
-      inputValue: null,
+      inputValue: null as any,
```
Comment (sh7dm) [inline packages/vuetify/src/components/VStepper/VStepper.ts]: Maybe `string | number | null` ?

File: packages/vuetify/src/components/VStepper/VStepper.ts [hunk 3]
```diff
@@ -74,30 +85,30 @@ export default {
...
       }
     },
-    stepClick (step) {
+    stepClick (step: string | number) {
       this.$nextTick(() => (this.inputValue = step))
     }
   },
 
-  render (h) {
+  render (h): VNode {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VStepper/VStepper.ts]: `h: Function` will be better

File: packages/vuetify/src/components/VStepper/VStepperStep.ts [hunk 4]
```diff
@@ -89,13 +107,13 @@ export default {
...
     },
-    toggle (step) {
+    toggle (step: number | string) {
       this.isActive = step.toString() === this.step.toString()
       this.isInactive = Number(step) < Number(this.step)
     }
   },
 
-  render (h) {
+  render (h): VNode {
```
Comment (sh7dm) [inline packages/vuetify/src/components/VStepper/VStepperStep.ts]: Also `h: Function` will be better

File: packages/vuetify/src/components/VStepper/VStepper.ts (hunk 3 above)
Comment (johnleider) [reply packages/vuetify/src/components/VStepper/VStepper.ts]: It should be VNode for proper typings

File: packages/vuetify/src/components/VStepper/VStepperStep.ts (hunk 4 above)
Comment (johnleider) [reply packages/vuetify/src/components/VStepper/VStepperStep.ts]: It should be VNode for proper typings

File: packages/vuetify/src/components/VStepper/VStepper.ts (hunk 2 above)
Comment (johnleider) [reply packages/vuetify/src/components/VStepper/VStepper.ts]: not a big deal, this is getting a full rewrite in 2.0

**Pull request merged**